M2Mqtt/project.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": "1.0.0-*",
+
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
+    "System.Collections.NonGeneric": "4.0.1",
+    "System.Net.NameResolution": "4.0.0",
+    "System.Net.Security": "4.0.0",
+    "System.Security.Cryptography.X509Certificates": "4.1.0",
+    "System.Threading.Thread": "4.0.0"
+  },
+
+  "frameworks": {
+    "netstandard1.6": {
```

Comment:
Why your code depends on netstandard1.6 instead of its previous versions? Now the code is not compatible with previous versions of netstandard.